projects/Rockchip/patches/linux/default/linux-0006-for-libreelec-h96mp.patch

```diff
@@ -0,0 +1,476 @@
+From d26b0d9e2e4e0ed22222b3c818314a2fbeeb9a1e Mon Sep 17 00:00:00 2001
+From: Ezra Buehler <ezra@easyb.ch>
+Date: Mon, 24 Feb 2020 23:46:05 +0000
+Subject: [PATCH] arm64: dts: rockchip: add H96 Max+ TV box support
+
+The device tree is based on DTS files for similar devices supported by
+LibreELEC and the DTB extracted from the H96 Max+ Android 9 image
+(RK3328_DC_H96_Max+_HS2734C_9_20191031.2215.img).
+
+Unfortunately, devfreq is not available for the GPU (lima dirver) yet.
+As a workaround, we simply do not configure pwm1 and the corresponding
+regulator.
+
+Signed-off-by: Ezra Buehler <ezra@easyb.ch>
+---
+ arch/arm64/boot/dts/rockchip/Makefile         |   1 +
+ .../boot/dts/rockchip/rk3328-box-h96mp.dts    | 435 ++++++++++++++++++
+ 2 files changed, 436 insertions(+)
+ create mode 100644 arch/arm64/boot/dts/rockchip/rk3328-box-h96mp.dts
+
+diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
+index c475e55a888c..3dac42051486 100644
+--- a/arch/arm64/boot/dts/rockchip/Makefile
++++ b/arch/arm64/boot/dts/rockchip/Makefile
+@@ -4,6 +4,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3308-evb.dtb
+ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3308-roc-cc.dtb
+ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-a1.dtb
+ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3328-box.dtb
```

Comment:
@Kwiboo I noticed that these *.dtb files were removed with bc55a55cbbb91e3fbc38cd52e73436def957772d. Is there a special reason for this? Do you need help porting them?